i18n/sync_tx_src.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * @fileoverview
+ * Script to upload a source en.json file to a particular transifex project resource.
+ * Expects that the project and resource have already been defined in Transifex, and that
+ * the person running the script has the the TX_TOKEN environment variable set to an api
+ * token that has developer access.
+ */
+
+// Fail immediately if the TX_TOKEN is not defined
+if (!process.env.TX_TOKEN) {
+  console.error(new Error('You must be a Scratch Team member and set your TX_TOKEN to sync with Transifex'));
```

Comment:
Hmmm... looking at this again, this is pretty unfriendly because it doesn't allow us to even get to the "Usage" instructions. Maybe we should incorporate this into that message. It's also not clear in this message how you should supply `TX_TOKEN` — if we include the TX_TOKEN check in the "Usage" check below, we should specify that `TX_TOKEN` needs to be set as an environment variable.